tests/inputs/doros_shrink_data.py

```diff
@@ -0,0 +1,47 @@
+""" 
+Quick script to remove some data from the DOROS files, to make them smaller for the tests.
+"""
+from pathlib import Path
+import shutil
+import subprocess
+import h5py
+
+from turn_by_turn.doros import DEFAULT_OSCILLATION_DATA, N_ORBIT_SAMPLES, N_OSCILLATION_SAMPLES, OSCILLATIONS, POSITIONS
+
+file_in = "/afs/cern.ch/work/j/jdilly/dorostest/DOROS-2024-09-29_01_37_13_522358-NO_USER.h5"
```

Comment:
Don't like a hardcoded path here

Reply:
I know. But it's just a script. people will have to hardcode their own path to run it anyway